packages/components/Avatar/src/Avatar.styling.ts

```diff
@@ -187,9 +185,11 @@ export const stylingSettings: UseStylingOptions<AvatarProps, AvatarSlotProps, Av
           status: tokens.badgeStatus,
           position: 'absolute',
           outOfOffice: tokens.outOfOffice,
+          right: tokens.badgeX !== undefined ? tokens.badgeX : undefined,
```

Comment:
Which case has this? Is it size16 - what is happens if you don't check on this?

Reply:
BadgeX and BadgeY, when present override the default badge position picked from presenceBadge tokens. This check prevents unnecessary overrides.
Added in comments.